src/Aquifer.API/Endpoints/Resources/Content/Get/Endpoint.cs

```diff
@@ -150,19 +157,22 @@ await dbContext.ResourceContentVersionAssignedUserHistory
             })
             .ToListAsync(ct);
 
-        resourceContent.MachineTranslations = relevantContentVersion.MachineTranslations.Select(mt => new MachineTranslationResponse
-        {
-            Id = mt.Id,
-            ContentIndex = mt.ContentIndex,
-            UserId = mt.UserId,
-            UserRating = mt.UserRating,
-            ImproveClarity = mt.ImproveClarity,
-            ImproveConsistency = mt.ImproveConsistency,
-            ImproveTone = mt.ImproveTone,
-            HadRetranslation = mt.RetranslationReason != null,
-            Created = mt.Created
-        })
-            .ToList();
+        // Can this move into the IsDraft check below?
```

Comment:
Need to check what the front-end is doing here, and since I want to hotfix this, will save for another time.